### DISCUSSION TITLE
No puedo instalar el compilador

### DISCUSSION THREAD

**Discussion by ELVAVAS** [INGCOM-UNRN-P1/cursada-2023#14] [Preguntas y respuestas] [answered]
He descargado el compilador de muchas formas pero se descarga un comprimido sin ningun instalador como muestran los tutoriales que he visto

**Comment by TeoInMaghine** [comment, marked as answer]
Si no me equivoco no usa instalador el compilador, creería que siguiendo las instrucciones de la [wiki](https://github.com/INGCOM-UNRN-P1/cursada-2023/wiki/Instalaci%C3%B3n-de-las-herramientas#compilador) te debería funcionar. (en Windows)

**Reply by ELVAVAS** [reply]
Gracias, ya lo encontré!
